### DISCUSSION TITLE
Autohide?

### DISCUSSION THREAD

**Discussion by winteriscariot** [LBCrion/sfwbar#226] [Q&A]
I'm trying to get auto-hide to work, but it's not behaving how I expect. I've set the 'sensor' option in my layout section, but the bar never appears when I move my mouse to the monitor edge it's attached to

here's my layout config:

```
layout "tint2:start" {
}

layout "tint2:center" {
  sensor = 1000
  taskbar {
    action[3] = Menu "winops";    
    sort = false
    icons = true
    labels = false    
    group = true
    group style = "taskbar_group"
    title_width = 10
    group cols = 1
    group icons = false
    group labels = true
    group title_width = 30
  }
  grid {
    include("volume.widget");
  }
}

layout "tint2:end" {
}
```

does the sensor option not work in the position layout option (ie, "tint2:center")? do i have to set the sensor option at the total layout level, ie creating a block that's simply `layout "tint2"`? I did try that and it didn't work, though.

using latest git labwc

thanks for any hints that i might have missed!

**Comment by LBCrion** [comment]
Just to confirm,  are you using the latest git version or one of the
releases?

**Reply by winteriscariot** [reply]
Not on absolute latest, but e2c5b19 which is pretty recent

**Reply by winteriscariot** [reply]
updated to latest git just cuz it only takes a minute, same behavior

**Comment by LBCrion** [comment]
I just tested on my system and it worked as expected,  so something must
differ in our setups. Can I check what compositor you're using? Also which
edge is your base on and is there another monitor in the same direction?
(I. E. If the bar is at the bottom is there another monitor below the
monitor with the bar?)

**Reply by winteriscariot** [reply]
I'm using [labwc](https://github.com/labwc/labwc), I had been wondering if this might be an issue with the compositor itself. is there something specific that the compositor needs to support to get it to work?

i do have 4 monitors, but sfwbar is on the bottom-most monitor so there's nothing below it

**Comment by LBCrion** [comment]
Is sfwbar on the bottom edge of the screen? (i.e. is this where it appears
if you comment out the 'sensor =' line?) This shouldn't be an issue with
labwc, I tested with it and it's working fine on my machine, so it should
be something else.

**Reply by winteriscariot** [reply]
It is, but I think I figured it out: there were some missing `=` signs in my config, which I think were an artifact of an earlier config file that hadn't been updated (i had some deprecated options like SetMonitor still in there). i also think setting layer to 'auto' may have helped.

anyway I've got a working config now, thanks for all your help!!
